src/core/signals.pm6

```diff
@@ -1,4 +1,10 @@
-my enum Signal (
+my role Signally {
+    multi method CALL-ME(Int $signum) {
```

Comment:
But this won't cover other reasonable cases like `Signal(‘0’)`?

Reply:
Thanks. I've pushed up a new commit that allows for scenarios like this.